rinex/src/navigation/health/gal.rs

```diff
@@ -1,88 +1,84 @@
-//! GAL Sv Health specifications
-use bitflags::bitflags;
+//! GAL SV Health specifications
 
-/// GAL Sv Health indication
-#[derive(Debug, Clone, PartialEq, PartialOrd)]
+/// GAL INAV & FNAV SV Health indication.
+/// See [Bibliography::RINEX3] for more information.
+#[derive(Default, Debug, Clone, PartialEq, PartialOrd)]
 #[cfg_attr(feature = "serde", derive(Serialize))]
-pub enum GALHealth {
-    /// GAL Legacy Sv Health indication
-    LNAV(LegacyHealth),
-    /// GAL INAV Sv Health indication
-    INAV(Health),
-    /// GAL FNAV Sv Health indication
-    FNAV(Health),
+pub struct GALHealth {
```

Comment:
I changed to a struct (with to/from `u32` conversions) as HS is actually a 2 bit integer and not two separate bit flags

Reply:
that's a good idea